box.json.dist

```diff
@@ -0,0 +1,56 @@
+{
+    "chmod": "0755",
+    "directories": [
+        "src"
+    ],
+    "files": [
+        "LICENSE"
+    ],
+    "finder": [
+        {
+            "name": [
+                "*.php"
+            ],
+            "exclude": [
+                "tests",
+                "Tests",
+                "phpunit/phpunit",
+                "phpunit/php-code-coverage",
+                "phpunit/phpunit-mock-objects",
+                "sebastian/comparator",
+                "sebastian/environment",
+                "sebastian/exporter",
+                "sebastian/global-state",
+                "sebastian/object-enumerator",
+                "sebastian/recursion-context",
+                "myclabs/deep-copy",
+                "phpspec/prophecy",
+                "phpdocumentor/reflection-docblock",
+                "phpdocumentor/type-resolver",
+                "phpdocumentor/reflection-common",
+                "webmozart/assert",
+                "phpunit/php-token-stream",
+                "sebastian/code-unit-reverse-lookup",
+                "phpunit/php-file-iterator",
+                "phpunit/php-text-template",
+                "doctrine/instantiator",
+                "sebastian/diff",
+                "sebastian/resource-operations",
+                "sebastian/version",
+                "phar-io/manifest",
+                "phar-io/version",
+                "theseer/tokenizer",
+                "sebastian/object-reflector"
+            ],
```

Comment:
List compiled by `composer update` and then reading off the removed packages running `composer update --no-dev` which presumably tells you exactly what is NOT needed to be included in distributable PHARs.